dts/msm8916/msm8939-xiaomi-ido.dts

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	qcom,msm-id = <0xef>;
```

Comment:
the id field is 2 cell long, should have kept the `<0xef 0x30000>` fully, also please decimal for the first cell
```suggestion
	qcom,msm-id = <239 0x30000>;
```